install.sh

```diff
@@ -42,8 +42,8 @@ esac
 sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
 # Delete 'var' symlink on Cloudready platform
 if [[ $(grep neverware /etc/lsb-release) != "" ]]; then
-  sudo unlink /usr/local/var
-  sudo unlink /usr/local/local
+  [ -L /usr/local/var ] && sudo rm /usr/local/var
+  [ -L /usr/local/local ] && sudo rm /usr/local/local
```

Comment:
Without the `-f` switch, you could get prompted.   I don't like prompts myself.

Reply:
Added